madgraph/iolibs/export_v4.py

```diff
@@ -7456,6 +7464,7 @@ def create_couplings_main(self, nb_def_by_file=25):
         fsock.writelines("""logical updateloop
                             common /to_updateloop/updateloop
                             include \'input.inc\'
+                            include \'../vector.inc\'
                             include \'coupl.inc\'
```

Comment:
same (again one which is less clear if we need it or not)

Reply:
same as above, vector.inc is needed by coupl.inc